camera_nodelet:
  camera_freq: 10.0  # Frequency of the camera (Hz)
  fixed_frame: "world"
  robot_frame: "base_link"
  camera_frame: "camera"
  fov_color: [0.0, 0.5, 0.5, 0.5]  # color of the camera field of view Rivz marker

  focal_length: 2.8e-3    # focal length of the camera
  sensor_width: 6.4e-3    # width of the camera sensor
  sensor_height: 3.84e-3  # height of the camera sensor
  fov_distance: 2.0  # maximum distance detected by the camera
  n_pxl_height: 10  # nbr of pixels along sensor height
  n_pxl_width: 20   # nbr of pixels along sensor width

  noise_meas: true    # whether to noise measurements on disease heatmap
  noise_std: 1.0      # maximum standard deviation of the measurement noise
  noise_decay: 1.0    # spatial decay rate for measurement noise standard deviation
